rele/publishing.py

```diff
@@ -41,5 +43,10 @@ def myfunc():
     :return: None
     """
     if not _publisher:
-        raise ValueError("init_global_publisher must be called first.")
+        settings, _ = discover.sub_modules()
```

Comment:
I know this will work for settings that are in the top level of the project dir. However, I have my doubts that it'll work for settings that are located elsewhere. Do you know if this is the case?

Reply:
Yeah, but I undestood this as a no boilerplate publish for simple setups. For a hidden settings inside the project it should be used with the config.setup. But we can change el discover to find settings inside the project.